### PR TITLE
Fixed bug with favorites

### PR DESCRIPTION
**Pull request by KseniiaEfremova**
1. Created a backend to store added favourites.
2. Bugs fixed according to comments: function renamed and URL constants created

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -43,18 +45,22 @@ function App() {
...
-      if (!prev.some(item => item.id === obj.id)) {
-        // If it doesn't exist, add it to the favorites array
-        return [...prev, obj];
-      }
-      // If it already exists, return the unchanged favorites array
-      return prev;
-    });
 
-  };
+  const onAddToFavorite = async (obj) => {
```
Comment (bendyna) [inline src/App.js]: looks like function name does not reflect its logic
It's not only adding, it's adding or deleting. Better to use verb "toggle"

File: src/App.js [hunk 2]
```diff
@@ -43,18 +45,22 @@ function App() {
...
+    if (obj.name) {
+      try {
+        if ((favorites).find(item => Number(item.id) === Number(obj.id))) {
+          axios.delete(`https://64707a693de51400f7245522.mockapi.io/favorites/${obj.id}`);
```
Comment (bendyna) [inline src/App.js]: let's move https://64707a693de51400f7245522.mockapi.io to const instead of copying

**Review by bendyna — CHANGES_REQUESTED** [review]
Please, give more descriptive names to commit and PR. For example, "Added favorites toggling".
Also, if you fixed bug, please add to PR description, how bug can be reproduced

File: src/App.js (hunk 2 above)
Comment (KseniiaEfremova) [reply src/App.js]: done

**Review by bendyna — APPROVED** [review]

**Pull request merged**